src/main/java/wooteco/subway/ui/LineController.java

```diff
@@ -0,0 +1,55 @@
+package wooteco.subway.ui;
+
+import java.net.URI;
+import java.util.List;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import wooteco.subway.dto.LineRequest;
+import wooteco.subway.dto.LineResponse;
+import wooteco.subway.dto.LineUpdateRequest;
+import wooteco.subway.service.LineService;
+
+@RestController
+public class LineController {
+
+    private final LineService lineService;
+
+    public LineController(LineService lineService) {
+        this.lineService = lineService;
+    }
+
+    @PostMapping("/lines")
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
+        return ResponseEntity.created(URI.create("/lines")).body(lineService.create(lineRequest));
+    }
+
+    @GetMapping(value = "/lines/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<LineResponse> showLine(@PathVariable Long id) {
+        return ResponseEntity.ok(lineService.showById(id));
```

Comment:
```suggestion
    public LineResponse showLine(@PathVariable Long id) {
        return lineService.showById(id);
```
응답 상태 코드가 200 OK라면 이런 식으로도 나타낼 수 있습니다.

Reply:
ResponseEntity 객체가 생성되는 비용을 아끼고 좀 더 간결한 코드가 되기때문일까요??

Reply:
이렇게도 나타낼 수 있다 정도의 코멘트로 받아들여주시면 좋을 것 같습니다!